package.json

```diff
@@ -2,22 +2,18 @@
   "name": "@lumigo/opentelemetry",
   "version": "1.19.1",
   "description": "Lumigo wrapper to trace distributed architecture",
-  "main": "dist/src/wrapper.js",
+  "main": "dist/src/distro.js",
   "private": false,
-  "types": "dist/src/wrapper.d.ts",
+  "types": "dist/src/bootstrap.d.ts",
```

Comment:
Delicious life-hack to keep TS compatibility for end-users, without loading OpenTelemetry dependencies in the bootstrap until we know the Node.js version is safe.